src/dackar/similarity/utils.py

```diff
@@ -0,0 +1,21 @@
+
+def combineListsRemoveDuplicates(list1, list2):
+  """combine two lists and remove duplicates
+
+  Args:
+      list1 (list): the first list of words
+      list2 (list): the second list of words
+
+  Returns:
+      list: the combined list of words
+  """
+  combinedList = list1 + list2
+  seen = set()
+  result = []
+
+  for item in combinedList:
+      if item not in seen:
+          seen.add(item)
+          result.append(item)
+
+  return result
```

Comment:
is the for loop necessary? it can be reduced to:
combinedList = list1 + list2
result = list(set(combinedList))

Reply:
Unfortunately, we need to preserve the order also. Which means we can not use 'set'